lld/wasm/InputChunks.cpp

```diff
@@ -167,6 +167,20 @@ void InputChunk::relocate(uint8_t *buf) const {
   }
 }
 
+static bool relocIsLive(const WasmRelocation &rel, ObjFile *file) {
+  return rel.Type == R_WASM_TYPE_INDEX_LEB ||
+         file->getSymbol(rel.Index)->isLive();
+}
+
+size_t InputChunk::getNumLiveRelocations() const {
+  size_t result = 0;
+  for (const WasmRelocation &rel : relocations) {
+    if (relocIsLive(rel, file))
+      result++;
+  }
+  return result;
```

Comment:
Is there a modern C++ way to write an accumulator like this?

Reply:
I think the alternative would be something like
```
  return std::count_if(relocations.begin(), relocations.end(),
                       [&](const WasmRelocation &rel) {
                         return relocIsLive(rel, file);
                       });
```
That's... a little shorter, I guess?